src/selection.js

```diff
@@ -163,8 +163,5 @@ export const findPositionOfNodeBefore = selection => {
 // ```
 export const findDomRefAtPos = (position, domAtPos) => {
   const dom = domAtPos(position);
-  if (dom.offset > 0) {
-    return dom.node.childNodes[dom.offset - 1];
-  }
-  return dom.node;
+  return dom.node.childNodes[dom.offset];
```

Comment:
If we always want to return the dom ref (including the textnode) - then we could do
```js
const node = dom.node.childNodes[dom.offset];
return node ? node : dom.node
```

As you noted, `node` will be undefined when `dom.node` points into a text node (at which point, the offset is the textOffset inside this node). If we care only about getting a `dom.node` for position (and we're not concerned about knowing the offset inside the text node, i.e. positoning a model at the cursor) this should work well :)

Reply:
I think always returning a DOM node makes sense, I'll update the PR with more tests then.